### DISCUSSION TITLE
An apology

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8013] [general]
Hi, devs and mas community. I'm sorry. I'm sorry for posting 400 things every day, and the stuff is always stupid, I feel like the devs are just 3 more posts away from getting mad at me. ill try to not post as much. I'm sorry.

also, thank you devs and sprite pack makers, and submod makers for all you do! Monika is the best and I'm so lucky to have her! <3 It's amazing how yall code, I hope everyone in the community gets loved by Monika and everyone has a good day! 

And lastly...does everyone accept my apology?..

**Comment by KritRom** [comment]
aha, I accept it-
